PubSub/src/Message.php

```diff
@@ -51,8 +51,20 @@ class Message
     private $subscription;
 
     /**
-     * @param array $message See
-     *        [PubsubMessage](https://cloud.google.com/pubsub/docs/reference/rest/v1/PubsubMessage).
+     * @param array $message {
+     *     Message Options
+     *
+     *     See [PubsubMessage](https://cloud.google.com/pubsub/docs/reference/rest/v1/PubsubMessage).
+     *
+     *     @type string $data The message data field. If this is empty, the message must
+     *           contain at least one attribute.
+     *     @type array $attributes Optional attributes for this message
+     *     @type array $messageId ID of this message, assigned by the
+     *           server when the message is published.
+     *     @type \DateTimeInterface $publishTime The time at which the message
```

Comment:
The `publishTime` method below takes this input and turns it into a `DateTimeImmutable`, should this actually be a string instead?